src/script/page/MainContent/panels/Collection/Collection.test.tsx

```diff
@@ -101,6 +101,9 @@ describe('Collection', () => {
   });
 
   it('displays collection details when a section is selected', async () => {
+    const imageMessages = new Array(14).fill(null).map(() => createImageMessage());
```

Comment:
Move 14 to enum/const :)

```suggestion
    const imageMessages = new Array(14).fill(null).map(createImageMessage);
```

Reply:
Also maybe create util for creating array with items :)
And have usage fn(length, callback) :)

Reply:
Added variable, simplified callback passed to map. We're creating image messages this way only once, creating a util is too much for a single test.